app/lib/forms/qualified_teacher_check.rb

```diff
@@ -70,14 +75,7 @@ def next_step
 
         return :check_answers if changing_answer?
 
-        if wizard.query_store.inside_catchment?
-          return :find_school if wizard.query_store.works_in_school?
-          return :kind_of_nursery if wizard.query_store.works_in_childcare?
-
-          return :your_employment
-        end
-
-        :choose_your_npq
+        :provider_check
```

Comment:
100%

The saves etc. all feel like they should be after_save/before_save steps, not in the next step. I didn't do any major refactoring because this question is going to be entirely deleted and I wasn't sure if it was worth getting into the weeds on it based on that.